src/assets/scss/base.scss

```diff
@@ -113,3 +113,350 @@ html {
 .cursor-default {
   cursor: default; /* arrow */
 }
+
+.bg-grey-1 {
```

Comment:
Are these all being used, or can some of them be deleted?

Reply:
bg-grey-1 is been used. bg-grey-2 to bg-grey-5 not used

Reply:
OK, please clean up (delete) what you can.
(But don't delete SCSS variables... we might still use those.)